src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemLink.cshtml

```diff
@@ -1 +1 @@
-﻿<a href="@(Model.Href ?? "#")">@Model.Text</a>
+<a href="@(Model.Href ?? "#")" @(!string.IsNullOrEmpty(Model.Target) ? $"target=\"" + Model.Target + "\"" : string.Empty)>@Model.Text</a>
```

Comment:
I think you should encode `Model.Target`

Reply:
encode ?  sorry, I don't quite understand what you mean. 😅

Reply:
To ensure the string is html encoded using something like this
```
Html.Encode(Model.Target)
```

Reply:
why？ 😳

Reply:
if you are not validating the user's input and just injecting string into HTML, then you have a risk of bad HTML. Like what if the user provided the following value?

```
some " bad value.
```

Reply:
As I recall it is automatic , unless you call @Html.Raw(Model.Target)

Reply:
I don't think it is automatic when you are doing string concatenation. But I think you are no longer doing string concatenation.

Reply:
I am resolving this. Please look and comment at the other comment above.